### ISSUE TITLE
Include note about accessibility of red/green sticky notes

### ISSUE THREAD

**Issue by angela-li** [carpentries/instructor-training#1032]
I mentioned this on the Carpentries Slack, but it is important to make sure colored sticky notes are accessible for helpers and teachers. I was recently teaching a workshop where the host had provided red and green sticky notes, but one of our helpers was colorblind and thus could not tell when people needed help. (Can you imagine if a learner had the same issue and could not signal for help in the way we asked them to?)

A note should be included in the [section introducing sticky notes](https://carpentries.github.io/instructor-training/14-live/#sticky-notes) explicitly mentioning this accessibility issue, and suggest different colors / different shapes / different use of sticky notes in order to make the workshop accessible to both teachers and learners who have colorblindness.

**Comment by amyehodge** [comment]
I have also had instances where the two colors we had were distinguishable, but not obvious which was the "all good" and which the "need help." When this happens, people tend to put a happy face on one and a sad face on the other or some other sign to themselves about which is which, and I think this works well. 

Alternatively, how fun would it be to have these and just hand out one that could be turned around? https://www.amazon.com/Eagle-100-Sheet-Sticky-Notes/dp/B01AHUR4SI

**Comment by angela-li** [comment]
I've created a pull request to include a note about this in the training, so please take a look and merge if it looks good to you!

**Comment by ChristinaLK** [comment]
This is a great point @angela-li and something that often comes up in discussion but wasn't yet in the materials.

**Comment by kariljordan** [comment]
So happy for this contribution @angela-li!
